events/wc_offmap_events.txt

```diff
@@ -629,30 +629,14 @@ character_event = {
 		clr_character_flag = requested_offmap_power
 	}
 }
-
 character_event = {
 	id = WCOME.100
-	desc = {
-       trigger = { has_character_modifier = follower_of_cthun }
-       text = EVTDESC_WCOME.100
-       picture = GFX_evt_cthun
-    }
-	desc = {
-       trigger = { has_character_modifier = follower_of_yogg_saron }
-       text = EVTDESC_WCOME.100
-       picture = GFX_evt_yogg_saron
-    }
-	desc = {
-       trigger = { has_character_modifier = follower_of_nzoth }
-       text = EVTDESC_WCOME.100
-       picture = GFX_evt_nzoth
-    }
-	border = GFX_event_normal_frame_religion
-	
+	hide_window = yes
 	is_triggered_only = yes
 	
 	trigger = {
 		religion = old_gods_worship
+		has_character_flag = hidden_sacrifice_on_siege
```

Comment:
You need `hidden_sacrifice_on_siege` in order to get this event, but you can't get it, since this flag will be set on character only in event that fires after this event

Reply:
fixed